src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4273,6 +4383,62 @@ components:
             validators:
               $ref: "#/components/schemas/ValidatorUptimeCollection"
 
+    #
+    # /wallet/* Request & Response Types
+    #
+
+    WalletTbdRequest:
+      allOf:
+        - $ref: "#/components/schemas/AtLedgerStateMixin" # TODO do we even want to support that? if yes, it could technically fall under regular /state/* endpoints
+        - type: object
+          required:
+            - lookup
+          properties:
+            lookup:
+              type: array
+              items:
+                $ref: "#/components/schemas/WalletTbdRequestLookupTuple"
+      example:
+        TBD
+    WalletTbdRequestLookupTuple:
+      allOf:
+        - $ref: "#/components/schemas/FromLedgerStateMixin"
+        - type: object
+          required:
+            - account_locker_address
+            - account_address
+          properties:
+            account_locker_address:
+              $ref: "#/components/schemas/Address"
+            account_address:
+              $ref: "#/components/schemas/Address"
+
+    WalletTbdResponse:
+      allOf:
+        - $ref: "#/components/schemas/LedgerStateMixin"
+        - type: object
+          required:
+            - items
+          properties:
+            items:
+              type: array
+              items:
+                $ref: "#/components/schemas/WalletTbdResponseItem"
+    WalletTbdResponseItem:
+      type: object
+      required:
+        - account_locker_address
+        - account_address
+        - resource_last_stored_at_state_version
+      properties:
+        account_locker_address:
```

Comment:
Decision: let's create separate key and value objects here, where `value` is optional and contains the required resource_last_stored_at_state_version - and we will return an entry for each key.

Reply:
Name of API will be decided in slack thread. I'd like to get that resolved this week. Russ to give sign-off if you need a go/no-go.